service/src/test/scala/za/co/absa/logingw/rest/service/InfoServiceTest.scala

```diff
@@ -32,6 +33,6 @@ class InfoServiceTest extends AnyFlatSpec with Matchers {
   new TestContextManager(this.getClass).prepareTestInstance(this)
 
   "InfoService" should "give expected test message" in {
-    infoService.getInfoMessage shouldEqual "Basic info message to be here."
+    infoService.getInfoMessage shouldEqual "Basic info message to be here. 'BETA'"
```

Comment:
Tests Should be able to parse the value from the config as well instead of hardcoding the value. This ensures that we don't have to change the tests every time we change the config and also helps if we introduce different config profiles for different environments.

Reply:
Test updated as you mentioned above. With the latest PR, now both tests are able to parse the value from the config instance.